Add new Kron4ek builds

jntesteves opened the pull request:
It was not tested locally because the documentation currently doesn't explain how to test components. If there's interest in merging this, I'll need help with testing. Based on the previous work by @keenanweaver.

## Type of change
- [x] New component
- [ ] Manifest fix
- [ ] Other

## Was this tested using a [Local Repository](https://maintainers.usebottles.com/Testing)?
- [ ] Yes
- [x] No

keenanweaver commented:
Thanks for continuing this! I hope it gets automated in the CI eventually as this Wine build gets updated frequently enough.

You can test it by using this command: `LOCAL_COMPONENTS=/path/to/repo flatpak run com.usebottles.bottles`

The terminal should say this: `07:43:22 (INFO) Using local components repository at /home/keenan/Documents/components/`

koplo199 commented:
@keenanweaver Done ! :partying_face: It is implemented in #234, but it should be merged after this one so it is marked as draft in the meantime

jntesteves commented:
> @keenanweaver Done ! partying_face It is implemented in #234, but it should be merged after this one so it is marked as draft in the meantime

Hey, that's cool. I don't see a reason why this should be merged before, when CI runs all these versions should be added by your script anyway, right?

koplo199 commented:
> Hey, that's cool. I don't see a reason why this should be merged before, when CI runs all these versions should be added by your script anyway, right?

No it unfortunately won't: the CI only checks for the most recent update

koplo199 commented:
@mirkobrombin Any chance to get this PR merged Soon™?

jntesteves commented:
I've added the new recent builds up to the just released proton-8.0-1. I've also changed the proton builds' sub-category from `proton` to `wine`, since these are not full proton builds, but just protonified wine from Valve's fork.

I'm thinking about dropping the 32-bit builds. Are these even usable in Bottles? They seem like unnecessary bloat, I don't see a use for then.

koplo199 commented:
I agree concerning 32-bit builds, they're probably not worth keeping.

jntesteves commented:
To document the exact script I used to generate these, here's the diff to bottlesdevs/tools: [https://github.com/jntesteves/bottlesdevs-tools/commit/5e0681b55cf4d384807818f251fe81088ac27a88](https://github.com/jntesteves/bottlesdevs-tools/commit/5e0681b55cf4d384807818f251fe81088ac27a88)

I removed the 32-bit builds like I said. I'm happy with this PR now, I think it's looking good to be merged. I've been personally using these, specially the new Proton 8 is amazing, it's now my default runner for games until Soda rebases on Proton 8. I think this offers quite some value to users of Bottles.

Bananaman commented:
> I've been personally using these, specially the new Proton 8 is amazing

Thank you for this. I wasn't aware of Kron4ek but I switched everything over to it now. It's way better than Valve's "junk", meaning Proton, GE-Proton and GE-Wine (which is still based on Proton).

With all Proton-based builds there have been long-standing issues such as the Minion app for Elder Scrolls Online not working. And memory injection from process A into another app process B doesn't work (almost as if Proton specifically removes that ability).

Those things broke after GE-Wine 7.2, about a year ago, when GE decided to base everything on Proton instead.

On a whim, I tried Kron4ek's Wine 8.6 Staging TKG. It's Wine 8.6 with staging and TKG patches. And uh, for the first time, everything works again. I don't think I'll ever move back to GE builds for as long as they stay based on Valve's Proton.

There's no doubt that Valve does amazing work for the Linux community, but Proton and all derivatives should stay where they belong - inside Steam.
